src/main/java/com/theironyard/savingprogress/Main.java

```diff
@@ -0,0 +1,81 @@
+package com.theironyard.savingprogress;
+
+import jodd.json.JsonParser;
+import jodd.json.JsonSerializer;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Created by darionmoore on 12/8/16.
+ */
+public class Main {
+    public static void main(String[] args) throws IOException {
+
+            Scanner input = new Scanner(System.in);
+            File file = new File("Hoodie.json");
+        try {
+            Scanner scanner = new Scanner(file);
+            scanner.useDelimiter("\\Z");
+            String contents = scanner.next();
+            JsonParser parser = new JsonParser();
+            Hoodie newHoodie = parser.parse(contents, Hoodie.class);
+            System.out.println(contents);
+        } catch (FileNotFoundException e){
```

Comment:
Good using the specific exception.  Though the goal would be only having the line that would throw this (`Scanner scanner = new Scanner(file);`) in the `try` block.